test/integration/targets/inventory_docker_machine/runme.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+# set DIGITALOCEAN_TOKEN to your Digital Ocean API token before running this test
+
+export DM_MACHINE_NAME=dm-test-machine
```

Comment:
I would suggest to randomize this name, to avoid conflicts when two tests are run at the same time:
```suggestion
export DM_MACHINE_NAME=dm-test-machine-${RANDOM}
```

Reply:
Thanks. I included this in my latest commit.